anxcloud/data_source_core_location.go

```diff
@@ -5,51 +5,31 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-	"go.anx.io/go-anxcloud/pkg/api"
-	"go.anx.io/go-anxcloud/pkg/api/types"
 	corev1 "go.anx.io/go-anxcloud/pkg/apis/core/v1"
 )
 
 func dataSourceCoreLocation() *schema.Resource {
 	return &schema.Resource{
-		Description: "Retrieves a location identified by it's `code` as selectable in the Engine. Use this data source to specify the location identifier on other resources and data sources available in this provider.",
+		Description: "Retrieves a location identified by it's `identifier` or `code` as selectable in the Engine. Use this data source to specify the location identifier on other resources and data sources available in this provider.",
```

Comment:
The usage example reads a bit weird now, given it says you can retrieve by identifier or code - maybe you find a better way to communicate that?

Could definitely stay this way, just wanted to point that out